docs/static/releasenotes.asciidoc

```diff
@@ -29,6 +31,72 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-6-0]]
+=== Logstash 8.6.0 Release Notes
+
+[[features-8.6.0]]
+==== New features and enhancements
+
+* Extends the flow rates introduced to the Node Stats API in 8.5.0 (which included windows for `current` and `lifetime`)
+to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
+https://github.com/elastic/logstash/pull/14571[#14571]
+* Logstash introduced instance and pipeline level flow metrics, growth_bytes and growth_events for persisted queue
+to provide a better visibility about how fast pipeline queue is growing each single seconds.
```

Comment:
```suggestion
* Logstash introduced instance and pipeline level flow metrics, `growth_bytes` and `growth_events` for persisted queue
to provide a better visibility about how fast pipeline queue is growing.
```

Reply:
The PR is just a forward port of #14762 which is already merged in `8.6`. 
I think that if we change her we should also backport then to the `8.6`.

Reply:
Ah if we need to change, we have to both...